.github/workflows/cmake.yml

```diff
@@ -11,10 +11,7 @@ on:
   workflow_dispatch:
   workflow_call:
 
-# cancel older, redundant runs of same workflow on same branch
-concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
-  cancel-in-progress: true
+# no concurrency constraints for callable workflows, set in caller instead
```

Comment:
i would have thought this remained in place?

Reply:
I must be deleted in the default branch before the twig from this branch can merge: https://github.com/openziti/ziti-tunnel-sdk-c/pull/1056#discussion_r1870504479